config/webpacker.yml

```diff
@@ -62,9 +62,6 @@ test:
   <<: *default
   compile: true
 
-  # Compile test packs to a separate directory
-  public_output_path: packs-test
```

Comment:
This was needed to get tests passing on CircleCI following JavaScript package upgrades being applied using `Yarn`